scripts/lib/CIME/SystemTests/irt.py

```diff
@@ -0,0 +1,56 @@
+"""
+Implementation of the CIME IRT. (Interum Restart Test)
+This test the model restart capability as well as the short term archivers interum restart capabilty
```

Comment:
typos: Should read:
`This tests the model's restart capability as well as the short term archiver's interim restart capability`